salt/modules/dockerng.py

```diff
@@ -5966,7 +5966,6 @@ def sls_build(name, base='opensuse/python', mods=None, saltenv='base',
 
     # start a new container
     ret = __salt__['dockerng.create'](image=base,
-                                      name=name,
```

Comment:
This was removed because there is no reason to add a name to the container when we are just going to delete it at the end.